### ISSUE TITLE
Cannot accept an invitation to event that does not contain primary email of the user

### ISSUE THREAD

**Issue by ravmike** [nextcloud/calendar#5023]
### Steps to reproduce

1. User0 creates an event and invites user1 to it
2. Forward the invitation mail to user2 
3. Try to accept invitation from user2 (or import an event to calendar)
4. Result: for user0, the invitation is displayed as accepted, user2 has no event in his calendar

### Expected behavior

The user2 has the event in the calendar (the second option is better because the events could 

### Actual behaviour

for user0, the invitation is displayed as accepted, user2 has no event in his calendar

### Calendar app version

4.2.3

### CalDAV-clients used

only NC

### Web server

None

### Database engine version

None

### PHP engine version

None

### Updated from an older installed version or fresh install

None

**Comment by ChristophWurst** [comment]
if user2 is not invited, they should not be able to end up in accepted state. they are referred to as "party crashers" IIRC

**Comment by tcitworld** [comment]
The invitation email is indeed made for a specific user, when accepting the invitation, user sessions are not set up and there's no check of the current session user. It's not sure if we could or should do something about it like https://www.rfc-editor.org/rfc/rfc5546#section-3.2.2.6 describes.

**Comment by ravmike** [comment]
It's interesting that the option "import event" for the *.ics file also doesn't work

**Comment by ChristophWurst** [comment]
Makes sense @tcitworld. I wrongly assumed the attendance of user2 would show as accepted but it's about user1, right @ravmike?

**Comment by ravmike** [comment]
Right

**Comment by ChristophWurst** [comment]
Got it. Then I do not see a lot we can improve other than adding a notice to the email that the invite link is personalized and shall not be forwarded.

**Comment by ravmike** [comment]
Sorry for a bit of absence 

Is there at least some way to import an event (*.ics file) to the calendar no matter who is the participant?
In practice, one can be a member of some e-mail group, and his address will not be in the event.

**Comment by SebastianKrupinski** [comment]
Hi @ravmike 

I just test imported a bunch of events that do not belong to the user and all worked fine.

So I'm going to close this ticket as no longer relevant.
